src/embed_tests/pysethome.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Python.Runtime;
+
+
+namespace Python.EmbeddingTest
+{
+    public class PySetHomeSet
+    {
+        [SetUp]
+        public void SetUp()
+        {
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            PythonEngine.Shutdown();
+        }
+
+        [Test]
+        public void TestSetHome()
+        {
+            string homePath = @"C:\Python27\";
+            PythonEngine.PythonHome = homePath;
+            PythonEngine.Initialize();
+            Assert.AreEqual(PythonEngine.PythonHome, homePath);
+        }
+    }
+}

```

Comment:
Updating this PR and noticed this comment. I documented the issue with `PythonEngine.PythonPath` on #414. All the `get` functions were broken on `64bit` python.